.travis.yml

```diff
@@ -17,13 +24,23 @@ matrix:
     - php: 7.1
     - php: 7.2
     - php: hhvm
-
-before_script:
-  - composer install
-
-script:
-  - vendor/bin/simple-phpunit
-
-branches:
-  only:
-    - master
+    - stage: release
+      php: 5.3
+      dist: precise
+      before_script:
+        - composer install --no-dev -o
+      script:
+        - curl -LSs https://box-project.github.io/box2/installer.php | php
```

Comment:
This one should be in `before_script` (or `install`), so that the next script does not run if downloading box failed (scripts are all running, even if the first one fails)